src/main/java/hudson/plugins/jira/selector/JqlIssueSelector.java

```diff
@@ -25,7 +25,6 @@ public class JqlIssueSelector extends AbstractIssueSelector {
 
     @DataBoundConstructor
     public JqlIssueSelector(String jql) {
-        super();
```

Comment:
Why?

Reply:
There is a checkstyle rule for not calling `super()` explicitly if it doesn't do anything (the defualt construstructor is implicitly called).